src/PortingAssistant.Client.Analysis/Utils/ApiCompatiblity.cs

```diff
@@ -14,6 +14,48 @@ public static class ApiCompatiblity
             RecommendedActionType = RecommendedActionType.NoRecommendation
         };
 
+        public static CompatibilityResult GetCompatibilityResult(PackageDetailsWithApiIndices package, CodeEntityDetails codeEntityDetails, string target = "netcoreapp3.1", bool checkLesserPackage = false)
+        {
+            //If invocation, we will try to find it in a later package
+            if (codeEntityDetails.CodeEntityType == CodeEntityType.Method)
+            {
+                return GetCompatibilityResult(package, codeEntityDetails.OriginalDefinition, codeEntityDetails.Package.Version, target, checkLesserPackage);
+            }
+            //If another node type, we will not try to find it. Compatibility will be based on the package compatiblity
+            else
+            {
+                var compatiblityResult = new CompatibilityResult
+                {
+                    Compatibility = Compatibility.UNKNOWN,
+                    CompatibleVersions = new List<string>()
+                };
+
+                if (package == null || !NuGetVersion.TryParse(codeEntityDetails.Package.Version, out var targetversion))
```

Comment:
No, the logic in API compatibility is looking for an alternate compatible package. We don't do that for other code entities because we don't have this information.